scalameta/parsers/js/src/test/scala/scala/meta/parsers/JSFacadeSuite.scala

```diff
@@ -36,7 +36,8 @@ class JSFacadeSuite extends FunSuite {
               "type" -> "Term.Name",
               "children" -> a(),
               "pos" -> pos(7, 11),
-              "value" -> "Main"
+              "value" -> "Main",
```

Comment:
Does it make sense to have value+syntax for names? I feel like it only makes sense for literals.